lm_eval/models/huggingface.py

```diff
@@ -372,7 +372,7 @@ def world_size(self):
 
     def _get_backend(
         self,
-        config: transformers.AutoConfig,
+        config: transformers.PretrainedConfig,
```

Comment:
Could we accommodate both `AutoConfig` and `PretrainedConfig`?

Reply:
This was just the IDE bugging me. Although:
```python
x = transformers.AutoConfig.from_pretrained("EleutherAI/pythia-160m")
isinstance(x, transformers.AutoConfig)
Out[14]: False
isinstance(x, transformers.PretrainedConfig)
Out[15]: True
```